performance.rst

```diff
@@ -4,200 +4,210 @@
 Performance
 ===========
 
-Symfony is fast, right out of the box. Of course, if you really need speed,
-there are many ways that you can make Symfony even faster. In this article,
-you'll explore some of the ways to make your Symfony application even faster.
+Symfony is fast, right out of the box. Of course, if you really need speed, the
+following performance checklists show how to make your application even faster.
 
-.. index::
-   single: Performance; Byte code cache
+Symfony Application Checklist
+-----------------------------
+
+#. :ref:`Install APCu Polyfill if your server uses APC <performance-install-apcu-polyfill>`
+#. :ref:`Enable APC Caching for the Autoloader <performance-autoloader-apc-cache>`
+#. :ref:`Use Bootstrap Files <performance-use-bootstrap-files>`
 
-Use a Byte Code Cache (e.g. OPcache)
-------------------------------------
+.. _performance-install-apcu-polyfill:
 
-The first thing that you should do to improve your performance is to use a
-"byte code cache". These caches store the compiled PHP files to avoid having
-to recompile them for every request.
+Install APCu Polyfill if your server uses APC
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-There are a number of `byte code caches`_ available, some of which are open
-source. As of PHP 5.5, PHP comes with `OPcache`_ built-in. For older versions,
-the most widely used byte code cache is `APC`_.
+If your production server still uses the legacy APC PHP extension instead of
+OPcache, install the `APCu Polyfill component`_ in your application to enable
+compatibility with `APCu PHP functions`_ and unlock support for advanced Symfony
+features, such as the APCu Cache adapter.
 
-.. tip::
+.. _performance-autoloader-apc-cache:
 
-    If your server still uses the legacy APC PHP extension, install the
-    `APCu Polyfill component`_ in your application to enable compatibility with
-    `APCu PHP functions`_ and unlock support for advanced Symfony features, such
-    as the APCu Cache adapter.
+Enable APC Caching for the Autoloader
```

Comment:
Maybe mention the commands `composer install --apcu-autoloader` and/or `composer dump-autoload --apcu` and link for the rest.
(I tried this in #8098)

`ApcClassLoader ` is deprecated (since 3.3), so I vote against recommending.